superset-frontend/src/addSlice/AddSliceContainer.tsx

```diff
@@ -41,6 +40,14 @@ export type AddSliceContainerState = {
 };
 
 const styleSelectContainer = { width: 600, marginBottom: '10px' };
+const StyledContainer = styled.div`
+  border-radius: ${({ theme }) => theme.gridUnit}px;
+  background-color: white;
```

Comment:
Let's use the theme color version of white... I think it's grayscale.light4 or something

Reply:
...light5. Off by 1 error ;)